.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -1,14 +1,33 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
+about: Propose an idea for a feature or an enhancement
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+<!--
+Before submitting a feature request, please take some time to read and consider
+how your idea aligns with the WordPress philosophy.
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+https://wordpress.org/about/philosophy/
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+In particular:
+
+- Is extra configuration required? WordPress should work out of the box.
+- Will the average user understand this? WordPress should be designed for the majority.
+- Are extra options required? WordPress should offer decisions, not options.
+- Will 80% of users appreciate and use this? (Can this instead be a plugin?) WordPress should remain clean and lean.
+- Does this improve ease of use? WordPress should strive for simplicity.
+-->
+
```

Comment:
A bit of brainstorming below here.

There are a bit many questions here. This is kind of thoughts to ponder questions before submitting a feature request.
 
Extra configuration and extra options is kinda the same.

Rephrasing....
Are additional configuration or options needed? If yes perhaps the feature request would initially be better as a plugin and then at a later time be considered for integration into Gutenberg.

"Will 80% of users appreciate and use this?"
That is a tricky question to answer.

Does this improve ease of use? = will the feature make Gutenberg easier to use?